roles/variables/vars/main.yml

```diff
@@ -189,17 +189,17 @@ kafka_controller_properties:
     properties:
       zookeeper.set.acl: 'true'
   broker_listener:
-    enabled: "{{ sasl_protocol=='plain' }}" # might need to add for kerberos also in later release
+    enabled: true
```

Comment:
why did this become true ?

Reply:
This was one of the tech debt in the system where with kip 966, Controller now also has to communicate with Broker i.e. Controller is client and broker is server. Earlier, only Broker had to communicate with Controller. Due to this change, Controller needs broker’s jaas config. This change was previously done for plain that is why the old condition but it needs to be done for all protocols.
https://confluentinc.atlassian.net/browse/ANSIENG-4244 (For kerberos)
https://confluentinc.atlassian.net/browse/ANSIENG-3230 (Implemented already)